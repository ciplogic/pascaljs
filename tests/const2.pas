program Const2;
const
  I: Integer = 123;
  S: String = 'hej';
  A: array[0..3] of Word = (1, 16, 256, 4096);
begin
  WriteLn('I = ', I);
  WriteLn('S = ', S);
  WriteLn('A[1] = ', A[1]);
end.