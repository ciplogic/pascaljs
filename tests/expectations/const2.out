I = 123
S = hej
A[1] = 16
